microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+
+from logging import Logger
+from typing import Any, Dict
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata, schema, search_space
+from lisa.node import Node
```

Comment:
The Node and simple_requirement should be imported from the lisa. If they are not in `lisa/__init__.py`, they should be. We hope test case writers can get all from `lisa.*`.